pages/index.js

```diff
@@ -212,57 +220,67 @@ export default class Index extends React.Component<Props, State> {
     return (
       <React.Fragment>
         <InitIntl raw={intl}>
-          {intlFull => (
-            <IntlProvider value={intlFull}>
-              <MetaHead
-                locale={language.iso}
-                currentPath={currentPath}
-                socialPhotos={socialPhotos}
-                otherMetaTagIds={Object.keys(cityData.otherMetaTags)}
-              />
-              <FetchedProvider value={fetched}>
-                <Menu
-                  lang={langId}
-                  isMobile={isMobile}
-                  cityTag={cityTag}
-                  isStopover={cityData.isStopover}
-                  usedLocales={usedLocales}
+          {intlFull => {
+            const translate = (key, value) =>
+              intlFull.translate(key, value).replace(/\[price\]/g, cityData.lowestPrice);
+
+            return (
+              <IntlProvider value={{ ...intlFull, translate }}>
+                <MetaHead
+                  locale={language.iso}
+                  currentPath={currentPath}
+                  socialPhotos={socialPhotos}
+                  otherMetaTagIds={Object.keys(cityData.otherMetaTags)}
                 />
-              </FetchedProvider>
-              <Hero logo={cityData.cityLogo} photo={cityData.mainPhoto} />
-              <StickyAction />
-              <Element name="slider">
-                <SliderSection
-                  sliderImages={sliderImages}
-                  areItineraries={Object.keys(cityData.itineraries).length !== 0}
+                <FetchedProvider value={fetched}>
+                  <Menu
+                    lang={langId}
+                    isMobile={isMobile}
+                    cityTag={cityTag}
+                    isStopover={cityData.isStopover}
+                    usedLocales={usedLocales}
+                  />
+                </FetchedProvider>
+                <Hero
+                  logo={cityData.cityLogo}
+                  photo={cityData.mainPhoto}
+                  departureForLowestPrice={cityData.departureForLowestPrice}
```

Comment:
why do we need these props in Hero?

Reply:
We don't, I forgot to remove them, will do that now.